sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -763,6 +721,12 @@
         ],
         "responses": {
           "200": {
+            "headers": {
+              "Link": {
+                "description": "next paginated result",
+                "type": "string"
+              }
+            },
             "description": "Tag details of a repository",
             "schema": {
               "$ref": "#/definitions/TagList"
```

Comment:
The TagList returned here holds a collection of `TagAttributesBase`, which we are not exposing publicly.  Per the[ .NET APIView](https://apiview.dev/Assemblies/Review/ba57bf904e524ce3bf647dad124ddc7c/12cb5d5471dd467da550ffe6b63c202d?doc=False), 

```C#
public virtual Pageable<TagProperties> GetTags(GetTagOptions options = null, CancellationToken cancellationToken = default);
```

We'll want this to return `TagProperties` flattened with `TagAttributesBase`

Reply:
@jeremymeng, if you'd like to see the work I did in C# language transforms on top of generated code to create the APIView, you can see that in this branch: https://github.com/annelo-msft/azure-sdk-for-net/tree/acr-experiment-5.  Please reach out if you have any questions or want to raise any concerns with things I did creating the prototype.  Thanks!

Reply:
Ok I think I got it now. The service returns something like
```
{
  repository: "...",
  tags: [
    { otherFields: "..." },
    { otherFields: "..." }
    ...
  ],
  link: "..."
}
```
And we want the shape of 
```
{
  tags: [
    { repository: "...", otherFields: "..." },
    { repository: "...", otherFields: "..." }
    ...
  ],
  link: "..."
}
```
I will investigate whether it is possible to achieve by changing swagger or doing transformation.

Reply:
This is hard to achieve using swagger transformation. We will need to do it in the convenience layer.